checks/alertmanager.go

```diff
@@ -63,7 +63,10 @@ func (c *AlertManagerChecker) Check(ctx *context.Context, extConfig external.Che
 
 	for alert := range alertMessage {
 		result := pkg.Success(check, ctx.Canary)
-		result.AddDetails(alertMessage[alert])
```

Comment:
I think we need to map in the labels and annotations - especially if we want to link alerts to nodes / namespaces / pods etc..

- We should also handle the starts at / ends at and status to map to check health